images/vscode-ssh-server/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# renovate
+FROM debian:stable-slim
```

Comment:
I think renovate will come along and pin this, which might be a lot of updates!  You should probably pin it to start now.

Reply:
Should just be one, right? I expected it to pin the digest and update that monthly. Debian doesn't have numeric versions, so I didn't want to pick bookworm/bullseye/etc...

Reply:
I was suggesting you pin it now so it's one less PR renovate will do

Reply:
Actually I'm not convinced now that it will. Shouldn't it have pinned the ubuntu base image for the kairos-ubuntu Dockerfile? We probably need to solve this for both Dockerfiles if not.